.devcontainer/devcontainer.json

```diff
@@ -37,7 +37,7 @@
 	"postCreateCommand": "sudo pip install -r requirements.txt",
 
 	// Use 'forwardPorts' to make a list of ports inside the container available locally.
-	// "forwardPorts": 8080,
+	 "forwardPorts": 8080,
```

Comment:
Great! We still need to tell the devcontainer to install the software necessary for the IBM cloud CLI. After ln 33:

`"ms-kubernetes-tools.vscode-kubernetes-tools"`